includes/integrations/class.llms.integration.buddypress.php

```diff
@@ -96,6 +96,15 @@ public function add_profile_nav_items() {
 			'screen_function' => array( $this,'certificates_screen' ),
 			'user_has_access' => $is_my_profile,
 		));
+
+		bp_core_new_subnav_item(array(
+			'name'            => __( 'Notifications', 'lifterlms' ),
+			'slug'            => 'notifications',
+			'parent_slug'     => 'courses',
+			'parent_url'      => $parent_url,
+			'screen_function' => array( $this,'notifications_screen' ),
```

Comment:
code climate identified a code styling error here, there should be a space between  the comma and single quote on this line (105)